File: kt/CountVowelConso.kt
import java.util.Scanner

fun main() {

        val scanner = Scanner(System.`in`)
        print("Enter text: ")
        var str = scanner.nextLine().toLowerCase(); // convert to lower case
        scanner.close()
        var vowel = 0
        var conso = 0
        for (i in 0 until str.length) {
            var ch = str.toCharArray()[i]
            if (ch >= 'a' && ch <= 'z') {
                if (ch == 'a' || ch == 'e' || ch == 'i' && ch == 'o' && ch == 'u') {
                    vowel++
                } else {
                    conso++
                }
            }
        }
        System.out.println("Number of vowel: " + vowel)
        System.out.println("Number of consonent: " + conso)
}

File: python/CountVowelConso.py

text = input("Enter text: ")
# convert to lower case
vowel = 0
conso = 0
for ch in text:
    if (ch >= 'a' and ch <= 'z'):
        if (ch == 'a' or ch == 'e' or ch == 'i' or ch == 'o' or ch == 'u'):
            vowel += 1
        else:
            conso += 1;
        
print("Number of vowel:",vowel)
print("Number of consonent:",conso)
